spec/support/factory_bot.rb

```diff
@@ -2,6 +2,8 @@
 
 # Require this here to get factories to play nice with engine
 require "factory_bot_rails"
+# https://github.com/stympy/faker
```

Comment:
I think to make our tests more meaningful we should set the locale of Faker (`Faker::Config.locale = "en-GB"`). So as we're both requiring it and setting some config we could do with creating a `faker.rb` file in the `spec/support/` folder

```ruby
# frozen_string_literal: true

# We use this to generate fake data. We believe it is more realistic to use
# nondeterministic data in tests as it better represents actual usage and is more
# likely to highlight issues in our app
require "faker"

Faker::Config.locale = "en-GB"
```